server/auth/types/basic/basic_auth.ts

```diff
@@ -104,17 +104,26 @@ export class BasicAuthentication extends AuthenticationType {
     response: LifecycleResponseFactory,
     toolkit: AuthToolkit
   ): KibanaResponse {
+    const nextUrlParam = composeNextUrlQeuryParam(
+      request,
+      this.coreSetup.http.basePath.serverBasePath
+    );
```

Comment:
Didn't find it used in `this.config.auth.anonymous_auth_enabled` section. Why need to take out? I would suggest put it into `else {` clause.

Reply:
sure, will do that